Compile Error on Ubuntu 18.04, driver c7

Hello! I am in the process of updating my MAS installation (standard updates to Ubuntu 18.04 seem to have broken the version I've been using). However after cloning this repository and running configure, the compiler reported the following error:

```
make[2]: Entering directory '/home/mce/src/mas/interfaces/mce_library'
cc -Wall -g -O2 -fPIC -fno-stack-protector -I ../../include -DMAS_PREFIX="\"/usr/mce\"" -DMAS_ETCDIR="\"/etc/mce\"" -DMAS_CONFDIR="\"\"" -DMAS_JAMDIR="\"/usr/mce/firmware\"" -DMAS_ROOTDIR="\"/usr/mce/mce_script\"" -DMAS_TEMPDIR="\"/tmp\"" -DMAS_DATAROOT="\"/data/cryo\"" -DMAS_DATADIR="\"current_data\""  -c -o acq.o acq.c
cc -Wall -g -O2 -fPIC -fno-stack-protector -I ../../include -DMAS_PREFIX="\"/usr/mce\"" -DMAS_ETCDIR="\"/etc/mce\"" -DMAS_CONFDIR="\"\"" -DMAS_JAMDIR="\"/usr/mce/firmware\"" -DMAS_ROOTDIR="\"/usr/mce/mce_script\"" -DMAS_TEMPDIR="\"/tmp\"" -DMAS_DATAROOT="\"/data/cryo\"" -DMAS_DATADIR="\"current_data\""  -c -o cmd.o cmd.c
cmd.c: In function ‘mcecmd_send_command_simple’:
cmd.c:296:5: error: expected ‘,’ or ‘;’ before ‘cmd’
     cmd.data[0] = 1;
     ^~~
<builtin>: recipe for target 'cmd.o' failed
```

I wonder if there's a semicolon missing on the line I reference below?

 https://github.com/multi-channel-electronics/mas/blob/942237fd15c0b320a6275201d0f4819c5099586d/interfaces/mce_library/cmd.c#L294

I'm going to try editing it locally and see if it works, and I'll report back.

Thanks!

--- a/interfaces/mce_library/cmd.c
+++ b/interfaces/mce_library/cmd.c
@@ -291,7 +291,7 @@ int mcecmd_send_command_simple(mce_context_t* context, int card_id, int para_id,
         .card_id  = (u16)card_id,
         .para_id  = (u16)para_id,
         .count    = 1
-    }
+    };
 
     cmd.data[0] = 1;
     cmd.checksum = mcecmd_cmd_checksum(&cmd);

--- a/interfaces/mce_library/dirfile.c
+++ b/interfaces/mce_library/dirfile.c
@@ -529,9 +529,9 @@ static int dirfile_destructor(mcedata_storage_t *storage)
     // Free the private data for the storage module, clear the structure
     dirfile_t *f = (dirfile_t*)storage->action_data;
     dirfile_free(f);
-    FREE_NOT_NULL(f)
+    FREE_NOT_NULL(f);
 
-        memset(storage, 0, sizeof(*storage));
+    memset(storage, 0, sizeof(*storage));
     return 0;
 }
 
@@ -654,9 +654,9 @@ static int dirfileseq_destructor(mcedata_storage_t *storage)
     // Free the private data for the storage module, clear the structure
     dirfileseq_t *f = (dirfileseq_t*)storage->action_data;
     dirfile_free(&f->active_dirfile);
-    FREE_NOT_NULL(f)
+    FREE_NOT_NULL(f);
 
-        memset(storage, 0, sizeof(*storage));
+    memset(storage, 0, sizeof(*storage));
     return 0;
 }
 

